src/index.js

```diff
@@ -81,20 +86,40 @@ class NetworkStateNotifier extends React.Component {
     });
   };
 
-  handleChecker = () => {
-    const { isOnline } = this.state;
+  toggleOnlineOffline = (status) => {
```

Comment:
this function sets the online status based `status` parameter. so it should be named with `set` prefix. It is not just toggling always.

Reply:
Ok I will rename it